src/Resources/contao/modules/ModuleLogin.php

```diff
@@ -111,13 +106,17 @@ protected function compile()
 
 		if (\System::getContainer()->get('session')->isStarted())
```

Comment:
If you are no longer using the flash bag, you don't need to verify that the session is started.

Reply:
Removed in 3ca51292380adaacf625919362b34c6d017fa398